src/main/java/com/cinema_booking/booking/movie/dto/MovieDTO.java

```diff
@@ -0,0 +1,21 @@
+package com.cinema_booking.booking.movie.dto;
+
+import java.util.Set;
+import java.util.UUID;
+
+import com.cinema_booking.booking.room.model.Room;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
```

Comment:
I wouldn't recommend using this annotation, as it carries many others inside. You should create the accordingly to your needs. https://projectlombok.org/features/